src/libslic3r/libslic3r.h

```diff
@@ -50,12 +50,12 @@ using coordf_t = double;
 // for a trheshold in a difference of radians,
 // for a threshold of a cross product of two non-normalized vectors etc.
 static constexpr double EPSILON = 1e-4;
-// Scaling factor for a conversion from coord_t to coordf_t: 10e-6
+// Scaling factor for a conversion from coord_t to coordf_t: 10e-5
 // This scaling generates a following fixed point representation with for a 32bit integer:
-// 0..4294mm with 1nm resolution
-// int32_t fits an interval of (-2147.48mm, +2147.48mm)
+// 0..42940mm with 10nm resolution
+// int32_t fits an interval of (-21474.84mm, +21474.84mm)
 // with int64_t we don't have to worry anymore about the size of the int.
-static constexpr double SCALING_FACTOR = 0.000001;
+static constexpr double SCALING_FACTOR = 0.00001;
```

Comment:
Decreasing the precision here is not preferable.
I have plan to support large coords, it's not prioritized atm.

Reply:
I'll see about reworking this to disable the button before the issue presents itself.

Reply:
> I'll see about reworking this to disable the button before the issue presents itself.

If this is in relation to the "add plate" button, note that disabling the button won't be a complete solution...  because changing the printer type, or changing the size of the build plate for the current printer, can also cause this fault even without changing the number of plates...